docs/developer/deploy/upgrade.rst

```diff
@@ -0,0 +1,57 @@
+Upgrade a Platform with the latest dev build
+============================================
+
+Prerequisites
+-------------
+- A Platform already installed with all pod up & running
+- A new metalk8s.iso image with a higher version
+
+Procedure
+---------
+
+If upgrading from a lower patch, minor or major version just follow the
+`standard upgrade procedure <https://metal-k8s.readthedocs.io/en/latest/operation/upgrade.html>`_.
+
+If upgrading from the same patch, minor and major version:
+
+1. Upload the new metalk8s.iso on the bootstrap node
+
+2. Locate how metalk8s.iso is mounted
+
+  .. code-block:: console
+
+    [root@zenkotda-bootstrap centos]# cat /etc/fstab | grep metalk8s.iso
+    /root/metalk8s.iso		/srv/scality/metalk8s-2.6.0-dev	iso9660	nofail,ro	0 0
+
+3. Unmount the current metalk8s.iso
+
+  ``umount /srv/scality/metalk8s-2.6.0-dev``
```

Comment:
I am not sure if it is really important to place this in a sphinx parse-literal so that the version prefix is updated for next MetalK8s 2.7 doc or so 

I mean something like:
```
.. parsed-literal::

   umount /srv/scality/metalk8s-|version|

```

Reply:
done